src/client/components/header/Header.js

```diff
@@ -1,24 +1,23 @@
 import React from 'react';
-import styles from './Header.module.css';
-import SearchContainer from '@root/client/components/header/search/SearchContainer'
+import Search from '@root/client/components/header/search/Search'
 import headerPicture from '@root/client/images/headerPicture.jpg';
+import cn from 'classnames';
+import './Header.module.css';
 
+export class Header extends React.Component {
 
-export const Header = (props) => {
-  return (
-    <div className={styles.header}>
+  render() {
+    return (
+      <div className="header">
+        <div className="header-netflixroulette">
```

Comment:
you could use here `<p>NETFLIX ROULETTE</p>`

Reply:
Done)